TedTalkMVVM/TedTalkMVVM/view/DetailTedTalkViewController.swift

```diff
@@ -37,10 +37,8 @@ class DetailTedTalkViewController: UIViewController{
         viewModel = viewModelDetail(tedTalkManager: DisplayManagerTedTalk.sharedTedTalkManager,
                                     observerTedTalk: { [unowned self] viewModel in
                                         self.showTedTalkDetail(talk: viewModel.getTedTalkDetailsDisplay())
-                                    },observerNextTedTalkButton: { [unowned self] viewModel in
-                                        self.buttonNextDisable()}, observerPreviousTedTalkButton: { [unowned self] viewModel in
-                                            self.buttonPreviousDisable()}, observerTedTalkButtons: { [unowned self] viewModel in
-                                                self.buttonsEnables()})
+                                    }, observerTedTalkButtons: { [unowned self] viewModelPrevious, viewModelNext  in
```

Comment:
just a minor comment, instead of naming those variables like viewModelPrevious it could be better to name them something like: previousButtonIsActive, it will be much more clear